### DISCUSSION TITLE
Start a private instance when PC is booted (debian)

### DISCUSSION THREAD

**Discussion by someones-alt** [searx/searx#2927] [General]
pretty much the title

**Comment by 0xhtml** [comment]
Take a look at the uwsgi setup page: https://searx.github.io/searx/admin/installation-uwsgi.html
It has some information on how to setup searx as a uwsgi service which can be simply enabled with `systemctl enable uwsgi@searx` to autostart on boot. (Assuming you are using systemd and haven't setup searx in docker)
